src/discussions/learners/LearnersView.test.jsx

```diff
@@ -247,4 +255,41 @@ describe('LearnersView', () => {
 
     expect(learners).toHaveLength(3);
   });
+
+  it('should display reported and previously reported message by passing activeFlags or inactiveFlags',
+    async () => {
+      await setUpLearnerMockResponse(2, 2, 1, ['learner-1', 'learner-2'], '', 1, 1);
+      await assignPrivilages(true);
+      await renderComponent();
+
+      await waitFor(() => container.querySelector('.text-danger'));
+
+      const reportedIcon = container.querySelector('.text-danger');
+
+      await act(async () => fireEvent.mouseEnter(reportedIcon));
+
+      const reported = await screen.getByText('2 reported');
+      const previouslyReported = screen.getByText('1 previously reported');
+
+      expect(reportedIcon).toBeInTheDocument();
+      expect(reported).toBeInTheDocument();
+      expect(previouslyReported).toBeInTheDocument();
+    });
+
+  it('should display load more button and display more learners by clicking on button.', async () => {
+    await setUpLearnerMockResponse();
+    await assignPrivilages(true);
+    await renderComponent();
+
+    await waitFor(() => container.querySelector('[data-testid="load-more"]'));
+
+    const loadMoreButton = container.querySelector('[data-testid="load-more"]');
```

Comment:
repetition of query here, this can be changed to const loadMoreButton = await container.querySelector('[data-testid="load-more"]');